app.js

```diff
@@ -66,10 +66,10 @@ app.get('/:uuid', (req, res) => {
       return;
     }
     logScanned(req.params.uuid, matches[0].fixture);
-    matches.similarItems = searchDatabase({ fixture: matches[0].fixture }, allItems)
+    matches[0].similarItems = searchDatabase({ fixture: matches[0].fixture }, allItems)
```

Comment:
if we always point to matches[0] maybe we dont need matches to be a list.

Reply:
If memory serves, I added the [0] to cut the duplicate UUIDs in the spreadsheet. If I remove it, any more of the duplicates will break the rendering.

Reply:
Understood